### PR TITLE
Update and relaunch browser comparison pages

### PR DESCRIPTION
**Pull request by craigcook**
## Description
Restoring the previously removed/redirected pages, migrating them all to Fluent, and updating copy and info. Also adding a new page for Brave.

My first time doing Fluent migrations so be gentle.

## Issue / Bugzilla link
#8811 

## Testing
http://localhost:8000/firefox/browsers/compare/
https://www-demo1.allizom.org/firefox/browsers/compare/

### CODE REVIEW

**Comment by craigcook** [comment]
This is awaiting stakeholder review to double-check for accuracy so it's still a do-not-merge for the moment, but we can start code review.

**Comment by alexgibson** [comment]
Oh, also don't forget to add `[skip l10n]` to the commit message here as the migration will be handled manually.

File: l10n/en/brands.ftl [hunk 1]
```diff
@@ -17,6 +17,7 @@
 -brand-name-mozilla-corporation = Mozilla Corporation
 -brand-name-mozilla-foundation = Mozilla Foundation
 -brand-name-twitter = Twitter
+-brand-name-github = GitHub
```
Comment (alexgibson) [inline l10n/en/brands.ftl:20]: Should these be in alphabetical order, to match what was added to https://github.com/mozilla-l10n/www-l10n/blob/master/en/brands.ftl ?

**Comment by alexgibson** [comment]
Approved and merge at will - although it may be a good idea to wait until the migration has been reviewed by Peiying, in the event that she suggests changes and you need to re-run it.

**Pull request merged**